ouroboros-network-protocols/src/Ouroboros/Network/Protocol/CBOR.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE RankNTypes #-}
+
+module Ouroboros.Network.Protocol.CBOR where
+
+import qualified Codec.CBOR.Decoding as CBOR
+import qualified Codec.CBOR.Encoding as CBOR
+import qualified Codec.Serialise as CBOR
+
+data CBORCodec' a b = CBORCodec {
+    encodeCBOR :: a -> CBOR.Encoding,
+    decodeCBOR :: forall s. CBOR.Decoder s b
+  }
+
+type CBORCodec a = CBORCodec' a a
+
+defaultCodec :: ( CBOR.Serialise a
+                , CBOR.Serialise b
+                )
+             => CBORCodec' a b
+defaultCodec = CBORCodec CBOR.encode CBOR.decode
```

Comment:
This name is misleading, as we are trying to avoid `Serialise` as much as possible, we probably use it only in testing, isn't it?

Reply:
which name exactly. That `Serialise` is from CBOR and not from `Codec.Serialise` which I believe is the one we are avoiding

Reply:
Change to `serialiseCodec`

Reply:
Done!